### PR TITLE
New issue template for bugs related to Nullable Reference Types

### PR DESCRIPTION
**Pull request by T-Gro**

### CODE REVIEW

**Comment by vzarytovskii** [comment]
We can also use `yml` template, it will render as form:
```yml
name: Nullness-related report
description: File a bug report.
title: "[Nullness]: "
labels: ["bug", "Needs-Triage", "Area-Nullness"]
body:
  - type: markdown
    attributes:
      value: |
        Please check at [Nullable Reference Types RFC](https://github.com/fsharp/fslang-design/blob/main/RFCs/FS-1060-nullable-reference-types.md) if this issue isn't a known limitation, such as missing flow-control analysis after branching constructs like `if` or `while`.
  - type: input
    id: description
    attributes:
      label: Description
      description: Please provide a succinct description of the issue
      placeholder: Description
    validations:
      required: true
  - type: checkboxes
    id: categories
    attributes:
      label: Choose one or more from the following categories of impact
      options:
        - label: Unexpected nullness warning (false positive in nullness checking, code uses --checknulls and langversion:preview).
        - label: Missing nullness warning in a case which can produce nulls (false negative, code uses --checknulls and langversion:preview).
        - label: Breaking change related to older `null` constructs in code not using the checknulls switch.
        - label: Breaking change related to generic code and explicit type constraints (`null`, `not null`).
        - label: Type inference issue (i.e. code worked without type annotations before, and applying the --checknulls enforces type annotatins).
        - label: C#/F# interop issue related to nullness metadata.
        - label: Other (none of the categories above apply).
    validations:
      required: true
  - type: dropdown
    id: os
    attributes:
      label: Operating System
      description: What operating system are you running?
      options:
        - Windows (Default)
        - Linux
        - macOS
      default: 0
    validations:
      required: true
  - type: dropdown
    id: dotnet-kind
    attributes:
      label: What .NET runtime/SDK kind are you seeing the issue on
      multiple: false
      options:
        - .NET SDK (.NET Core, .NET 5+)
        - .NET Framework
      default: 0
    validations:
      required: true
  - type: input
    id: dotnet-version
    attributes:
      label: .NET Runtime/SDK version
      placeholder: e.g. .NET 9 or .NET SDK 9.0.100

```

File: .github/ISSUE_TEMPLATE/nullness_issue.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+name: Nullness-related report
+about: Create a report related to nullable reference types handling
+title: ''
+labels:  [Bug, Needs-Triage,Area-Nullness]
+assignees: 'T-Gro'
+
+---
+
+
+Please check at [Nullable Reference Types RFC](https://github.com/fsharp/fslang-design/blob/main/RFCs/FS-1060-nullable-reference-types.md) if this issue isn't a known limitation, such as missing flow-control analysis after branching constructs like `if` or `while`.
```
Comment (psfinaki) [inline .github/ISSUE_TEMPLATE/nullness_issue.md:11]: Would be good to have a direct link to known issue later.

**Review by psfinaki — APPROVED** [review]
Nice, the impact categories could be a bit more succinct but that's okay.
I guess if we have many bugs we can also bring new labels for them.

File: .github/ISSUE_TEMPLATE/nullness_issue.md (hunk 1 above)
Comment (T-Gro) [reply .github/ISSUE_TEMPLATE/nullness_issue.md:11]: These are orthogonal, will keep both.
Known limitations: Intentionaly not done, that is the RFC link
Known issue: Will point to a GH query over issues, i.e. already reported issues.

**Comment by T-Gro** [comment]
> Nice, the impact categories could be a bit more succinct but that's okay. I guess if we have many bugs we can also bring new labels for them.

Well labels are maintained by us, not by reporters.
I will manage/add the labels as the issues keep coming in.

**Comment by T-Gro** [comment]
@vzarytovskii :
Semi-rendered form template available here:
https://github.com/dotnet/fsharp/blob/df89da83a79dd817b323c82d64457ce348fa0ff9/.github/ISSUE_TEMPLATE/nullness_issue.yml 

When creating a new issue, looks like this (personal repo link):
https://github.com/T-Gro/fsharp/issues/new?assignees=T-Gro&labels=Bug%2CNeeds-Triage%2CArea-Nullness&projects=&template=nullness-issue.yml&title=Nullness+issue+-+....

**Comment by T-Gro** [comment]
@vzarytovskii : Available here: https://github.com/dotnet/fsharp/blob/5c974952bdf6f68558d35483727dd474851c8202/.github/ISSUE_TEMPLATE/nullness_issue.yml

**Review by vzarytovskii — APPROVED** [review]

**Pull request merged**